### PR TITLE
add mention of steel to dev docs site

### PR DESCRIPTION
**Pull request by pdg744**
The dev docs site doesn't mention Steel yet. This PR adds a brief section to the Ethereum Examples page, to improve findability of Steel. 

Questions worth thinking about:
- is the risc0-steel crate is the appropriate place to point users
- do we want to add mentions/paths to Steel elsewhere (mb in the sidebar?)

### CODE REVIEW

**Comment by NukeManDan** [comment]
Weird that can see the preview, but I cannot see the changes on https://docs-website-git-paulg-mention-steel-risczero.vercel.app/api/blockchain-integration/eth-examples 

Also a 404 for access the `inspect` url https://vercel.com/risczero/docs-website/5weXuQhj5RLTJ7ykoxWN4oKT2n5X - maybe I need to be grated access @pdg744 ?

**Comment by NukeManDan** [comment]
Idea: Would it be appropriate to expand a bit on each example listed here here on the "why" of Them? The steel blog linked for view cals helps give me a bit more understanding:

> How is this different from storage proofs?
> With traditional storage proofs, developers must manually select which storage slots their smart contract uses, and re-implement smart contract logic. With steel, all storage slots are discovered and fetched automatically based on the view call execution. This saves developers a significant amount of time and decreases the room of implementation errors, reducing the chance of security vulnerabilities.

Overall I think this page could be a good way to succinctly duscuss not just pointers to what exists, but why each one of these is unique and important for EVM related integrations. :thinking:

**Review by NukeManDan — APPROVED** [review]
For getting minimal visibility this LGTM :+1:

File: website/api/blockchain-integration/eth-examples.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ While all of the [zkVM examples][zkvm-examples] can be run on Bonsai by [configu
 
 The [RISC Zero Foundry Template][foundry-template] provides a minimal application that can act as a template for developing your application.
 
+### Steel
+
+[Steel] is a library for [view call proofs], which enables users to easily integrate claims about Ethereum state into zkVM applications.
```
Comment (nategraf) [inline website/api/blockchain-integration/eth-examples.md:11]: I've been finding it helpful to use tags like this for links. It helps understand where the link goes when reading the Markdown text... and [my linter](https://github.com/DavidAnson/markdownlint) yells at me when the links break with this format, but not without the tag.
```suggestion
[Steel][steel-repo] is a library for [view call proofs][steel-blog], which enables users to easily integrate claims about Ethereum state into zkVM applications.
```

File: website/api/blockchain-integration/eth-examples.md [hunk 2]
```diff
@@ -23,3 +27,5 @@ This [example app][governance-example] uses Bonsai as an Ethereum coprocessor. T
 [Governor smart contract standard]: https://docs.openzeppelin.com/contracts/4.x/api/governance
 [signature-aggregation]: https://github.com/risc0/risc0/blob/release-0.20/bonsai/examples/governance/methods/guest/src/bin/finalize_votes.rs
 [foundry-template]: https://github.com/risc0/bonsai-foundry-template
+[Steel]: https://crates.io/crates/risc0-steel
+[view call proofs]: https://www.risczero.com/blog/introducing-steel
```
Comment (nategraf) [inline website/api/blockchain-integration/eth-examples.md:31]:
```suggestion
[steel-repo]: https://crates.io/crates/risc0-steel
[steel-blog]: https://www.risczero.com/blog/introducing-steel
```

**Review by nategraf — APPROVED** [review]

**Comment by pdg744** [comment]
> Idea: Would it be appropriate to expand a bit on each example listed here here on the "why" of Them? 

thanks for raising this idea! i think we should aim for keeping these pretty concise especially considering that this list is likely to grow. that said, this page, and our blockchain docs in general, could use some improvement. would be great for the docs to offer a clearer answer to the question of "where should i start?" and "where should i go next?" when it comes to blockchain development.

**Pull request merged**